src/runner/helpers/ignored_objects_path.rs

```diff
@@ -36,14 +36,26 @@ fn get_node_objects() -> Vec<String> {
         debug!("Failed to get node shared objects: {:?}", output.stderr);
         return vec![];
     }
-    let so_output = String::from_utf8_lossy(&output.stdout).trim().to_string();
-    vec![so_output]
+    let so_str = String::from_utf8_lossy(&output.stdout).trim().to_string();
+    vec![so_str]
+}
+
+fn normalize_objects_path(objects_path_to_ignore: &mut [String]) {
```

Comment:
As seen together, let's check that it does not break the currently working flamegraphs in node and python

Reply:
Python:
https://github.com/CodSpeedHQ/pytest-codspeed/pull/26
Node:
https://github.com/CodSpeedHQ/codspeed-node/pull/33